examples/examples_yaml_test.go

```diff
@@ -23,3 +24,40 @@ func TestYamlTeamsExample(t *testing.T) {
 		},
 	})
 }
+
+func TestYamlStackTagsExample(t *testing.T) {
+
+	// Set up tmpdir with a Pulumi.yml with no resources
+	// mimicking the deletion of resource
+	tmpdir := t.TempDir()
+	newProgram := map[string]string{
+		"name":        "yaml-stack-tags-example",
+		"runtime":     "yaml",
+		"description": "A minimal Pulumi YAML program",
+	}
+
+	b, err := yaml.Marshal(newProgram)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = os.WriteFile(path.Join(tmpdir, "Pulumi.yml"), b, 0666)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cwd, _ := os.Getwd()
+	integration.ProgramTest(t, &integration.ProgramTestOptions{
+		Quick:       true,
+		SkipRefresh: true,
+		Dir:         path.Join(cwd, ".", "yaml-stack-tags"),
+		StackName:   "test-stack",
+		PrepareProject: func(_ *engine.Projinfo) error {
+			return nil
+		},
+		EditDirs: []integration.EditDir{
+			{
+				Dir: tmpdir,
+			},
+		},
```

Comment:
Just to make sure I understand - the `Dir` sets your initial resources to the stuff you defined in `examples/yaml-stack-tags/Pulumi.yaml`, then the `EditDirs` is basically simulating an updating using the resource-free file you just wrote?

Reply:
That is correct, yes!